{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/config/settings/base.py

```diff
@@ -211,7 +211,7 @@
         # https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-TEMPLATES-BACKEND
         "BACKEND": "django.template.backends.django.DjangoTemplates",
         # https://docs.djangoproject.com/en/dev/ref/settings/#template-dirs
-        "DIRS": [APP_DIR / "templates"],
+        "DIRS": [APP_DIR / "dtl_templates"],
```

Comment:
The only problem I see with this is that now we need to check every time there is a django update if these files have changed. I can't think of a better way to divorce them though given the constraints

Reply:
Do we need to include the dtl_tempaltes dir in this PR as well?

Reply:
In all cases being adequately captured by this PR (not all but hopefully enough): 
- The DTL will be used instead of Jinja2
- DTL will look for templates defined locally
- By default (unless we add one), it will never find a locally defined template, because even the base dir it's looking in does not exist
- It will default back to the base django templates

specifically
re django update collisions: I don't think this makes us any more vulnerable to collisions than we would otherwise be
re adding the dir: It'll fail gracefully if the file isn't found: https://github.com/django/django/blob/d1855c4847215f3afe3708736be13388bb6437eb/django/template/loaders/filesystem.py#L20 (I did not include the dir in my test project)

Reply:
Oh sorry, I may have misinterpreted because of the line the comment was attached to -- If you're referring to the methods the decorator is overriding. yes absolutely that is vulnerable to the underlying django updating. I tried to minimize the surface area but it's a fundamentally hacky approach. EXTREMELY HAPPY to throw all of this away in favor of a less fragile approach.